assets/i18n/en.toml

```
@@ -106,21 +106,21 @@ remaining = "Remaining"
 [main.loadpointSettings]
 title = 'Settings "{0}"'
 vehicle = "Vehicle"
-currents = "Charging"
+currents = "Charging current"
 default = "default"
 disclaimerHint = "Note:"
-disclaimerText = "Changes are not persistent yet. They will be reset after server restart."
+disclaimerText = "Changes will be lost after the server is restarted."
 
 [main.loadpointSettings.minSoC]
-label = "Minimal SoC"
-description = 'Range for emergencies. Vehicle gets "fast" charged to {0}% in PV mode. Then continues with PV surplus only.'
+label = "Minimal state of charge"
```

```suggestion
label = "Min. charge %"
```
Meant to avoid "SoC" and "state of charge" entirely.